src/components/organisms/AdminVenueView/components/RunTabView/RunTabView.tsx

```diff
@@ -1,60 +1,137 @@
-
   return (
-    <AdminPanel variant="unbound" className="RunTabView">
+    <ThreeColumnLayout>
       <AdminSidebar>
-        <AdminSidebarSectionTitle>
-          Run your {SPACE_TAXON.lower}
-        </AdminSidebarSectionTitle>
-      </AdminSidebar>
-      <div className="RunTabView__body">
-        <div className="RunTabView__user-list">
-          <UserList
-            usersSample={usersSample}
-            userCount={userCount}
-            showTitle={false}
-            activity={`in ${SPACE_TAXON.lower}`}
-          />
-        </div>
-        <AdminShowcase>
-          <div className="RunTabView__map RunTabView--spacing">
-            <MapPreview
-              isEditing={false}
-              mapBackground={mapBackground}
-              rooms={rooms}
-              setSelectedRoom={() => undefined}
+        <form onSubmit={handleSubmit(updateVenue)}>
+          <SidebarHeader>Announcements</SidebarHeader>
+          <InputGroup title="Announcement content" isRequired withLabel>
+            <Textarea
+              placeholder="Please type your announcement text here."
+              register={register}
+              name="content"
+              required
             />
+          </InputGroup>
+          <Checkbox
+            label="Set full-screen announcement"
+            register={register}
+            name="isFullScreen"
+            disabled
+            checked={values.isFullScreen}
+          />
+          <Toggle
+            label="Call to action button"
+            register={register}
+            name="isActionButton"
+            checked={values.isActionButton}
+          />
+          {values.isActionButton && (
+            <>
+              <InputGroup title="Button text" isRequired withLabel>
+                <Input
+                  placeholder="Button display text"
+                  register={register}
+                  name="buttonDisplayText"
+                  required
+                />
+              </InputGroup>
+              <InputGroup title="Button URL" isRequired withLabel>
+                <Input
+                  placeholder="Button URL"
+                  register={register}
+                  name="buttonUrl"
+                  required
+                />
+              </InputGroup>
+            </>
+          )}
+
+          <Checkbox
+            label="Force funnel"
+            register={register}
+            name="isForceFunnel"
+          />
+          <div className="flex justify-end">
+            <Button
+              variant="secondary"
+              onClick={() => reset()}
+              disabled={!isDirty}
+            >
+              Cancel
+            </Button>
+            <Button loading={isUpdating} type="submit">
+              Save
+            </Button>
           </div>
-        </AdminShowcase>
-      </div>
-    </AdminPanel>
+        </form>
+      </AdminSidebar>
+
+      <AdminShowcase>
+        <div className="px-12 sm:px-12"></div>
```

Comment:
Is this `div` a placeholder for something that needs to be inserted in a latter PR? If so a comment of TODO might be appropriate or at least a comment explaining why it is empty

Reply:
Added a comment in https://github.com/sparkletown/sparkle/pull/2952/commits/38d6ac6a94dd7aa2549bf7fcb0ad9d2fc7a05a13